.github/workflows/test-helpers/wait-for-issue

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-function download-and-extract {
-  issue_number="$1"
+function download-and-extract-issue {
+  local issue_number="$1"
+
+  # I tried `gh issue view` but it does whack stuff with the terminal that makes
+  # it non-trivial to redirect/pipe. Pulling from JSON is probably more robust
+  # anyway.
+
+  gh api "/repos/:owner/:repo/issues/${issue_number}" > issue
+  jq -r .state issue > issue-state
+  jq -r .comments issue > issue-ncomments
+  gh api "/repos/:owner/:repo/issues/${issue_number}/comments" > issue-comments
+  jq -r '.[0].body' issue-comments > issue-comment
```

Comment:
Pro `jq` usage.

![doing it like a pro](https://media.giphy.com/media/eKIXKFcp4oibK/giphy-downsized.gif)